.github/workflows/test-use-cases.yml

```
@@ -73,4 +73,36 @@ jobs:
         payload: |
           {
             "text": "${{ env.stepreport }}"
-          }
\ No newline at end of file
+          }
+
+    - name: Report failure to Opsgenie
+      if: failure() && steps.check-enable.outputs.enabled == 'true' && steps.usecase-test.outcome == 'failure' && secrets.OPSGENIE_API_KEY != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)